### ISSUE TITLE
Limits of erfc

### ISSUE THREAD

**Issue by rtoy** [rtoy/maxima#963]
Imported from SourceForge on 2024-07-03 15:49:56
Created by **[mardukbp](https://sourceforge.net/u/mardukbp/)** on 2016-05-02 18:52:31
Original: https://sourceforge.net/p/maxima/bugs/3153

---

Input
~~~~
integrate(exp(-%i*ω^2*ρ^2-(%i*t^2)/(4*ρ^2)),ρ,0,inf);
~~~~

Output
~~~~
0
~~~~
Expected
~~~~
1/sqrt(%i/%pi)*1/(2*ω)*exp(-%i*ω*abs(t))
~~~~
Robert Dodier determined that there is a bug in
~~~~
limit(erfc(x*(1 + %i)), x, inf)
~~~~
It returns
~~~~
erfc(infinity)
~~~~
It should return
~~~~
'limit(erfc(x*(1 + %i)), x, inf)
~~~~
Additionally, Robert Dodier noted that

(1) This limit can be calculated with the substitution erfc -> 1 - erf

(2) Maxima needs help with `limit(erf(1/x*(1 + %i)), x, 0, plus)`. The substitution y = 1/x reduces this limit to the infinity case above.

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-03 15:49:58
Created by **[robert_dodier](https://sourceforge.net/u/robert_dodier/)** on 2016-05-02 23:33:25
Original: https://sourceforge.net/p/maxima/bugs/3153/#b075

---

- Description has changed:

Diff:

~~~~

--- old
+++ new
@@ -1,29 +1,30 @@
 Input
-
+~~~~
 integrate(exp(-%i*ω^2*ρ^2-(%i*t^2)/(4*ρ^2)),ρ,0,inf);
+~~~~
 
 Output
-
+~~~~
 0
-
+~~~~
 Expected
-
+~~~~
 1/sqrt(%i/%pi)*1/(2*ω)*exp(-%i*ω*abs(t))
-
+~~~~
 Robert Dodier determined that there is a bug in
-
+~~~~
 limit(erfc(x*(1 + %i)), x, inf)
-
+~~~~
 It returns
-
+~~~~
 erfc(infinity)
-
+~~~~
 It should return
-
+~~~~
 'limit(erfc(x*(1 + %i)), x, inf)
-
+~~~~
 Additionally, Robert Dodier noted that
 
 (1) This limit can be calculated with the substitution erfc -> 1 - erf
 
-(2) Maxima needs help with limit(erf(1/x*(1 + %i)), x, 0, plus). The substitution y = 1/x reduces this limit to the infinity case above.
+(2) Maxima needs help with `limit(erf(1/x*(1 + %i)), x, 0, plus)`. The substitution y = 1/x reduces this limit to the infinity case above.

~~~~

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-03 15:50:01
Created by **[robert_dodier](https://sourceforge.net/u/robert_dodier/)** on 2016-05-02 23:33:25
Original: https://sourceforge.net/p/maxima/bugs/3153/#aaa0

---

Fixed up formatting.

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-03 15:50:05
Created by **[dgildea](https://sourceforge.net/u/dgildea/)** on 2016-06-10 20:04:33
Original: https://sourceforge.net/p/maxima/bugs/3153/#bff8

---

- **status**: open --> closed
- **assigned_to**: Dan Gildea

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-03 15:50:09
Created by **[dgildea](https://sourceforge.net/u/dgildea/)** on 2016-06-10 20:04:33
Original: https://sourceforge.net/p/maxima/bugs/3153/#d90f

---

Limit of erfc fixed in commit [5b5e352].
Integral now returns an unevaluated limit.
